src/main/groovy/ga/ozli/minecraftmods/groovylicious/Groovylicious.groovy

```diff
@@ -1,21 +1,20 @@
 package ga.ozli.minecraftmods.groovylicious
 
+import com.matyrobbrt.gml.GMod
 import groovy.transform.CompileStatic
 import groovy.transform.PackageScope
 import groovy.transform.stc.POJO
-import net.thesilkminer.mc.austin.api.Mod
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
+import groovy.util.logging.Slf4j
 
 @POJO
+@GMod(MOD_ID)
 @CompileStatic
-@Mod(MOD_ID)
+@Slf4j(category = MOD_ID)
```

Comment:
![image](https://user-images.githubusercontent.com/3158390/185182639-bf5acb60-6854-4b7b-90ce-973dfedc9476.png)
I didn't do this as it breaks logging in prod

Reply:
That... doesn't seem right